src/util/transfer.ts

```diff
@@ -26,12 +26,12 @@ async function getAccInfo(address :string) :Promise<PublicAccount> {
 }
 
 
-export const createTransaction = async (recipient :string, sendXPX :string, messageText :string, mosaicsSent :{amount: number ,id :string}[], mosaicDivisibility :number[], walletPassword :string, senderAccAddress :string, selectedCosigner :string, cosignerList :{publicKey :string}[],encryptedMsg :string) : Promise<boolean>  => {
+export const createTransaction = async (recipient :string, sendXPX :string, messageText :string, mosaicsSent :{amount: number ,id :string}[], mosaicDivisibility :number[], walletPassword :string, senderAccAddress :string, selectedCosigner :string, encryptedMsg :string) : Promise<boolean>  => {
```

Comment:
please fix data type,
selectedCosigner should be `string | undefined`
encryptedMsg should be `boolean`